src/components/ViewMoreListBtn/ViewMoreListBtn.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { Box, Button } from "@mui/material";
+import ExpandMoreIcon from "@mui/icons-material/ExpandMore";
+import { useStyles } from "./styles";
+
+export default function ViewMoreListBtn({ onViewMore }: { onViewMore: () => void }) {
+  const styles = useStyles();
+  return (
+    <Box
+      className={styles.btnWrapper}
+      onClick={() => {
```

Comment:
Why are you using arrow function in the onClick handler?

Reply:
I thought it was the best practice after I took some course few months ago that it is better to call the function as arrow function. I will update this night. And other corrections you pointed out.